src/main/java/org/spoken_tutorial/health/elasticsearch/models/QueueManagement.java

```diff
@@ -0,0 +1,236 @@
+package org.spoken_tutorial.health.elasticsearch.models;
+
+import java.sql.Timestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+@Entity
+public class QueueManagement {
+
+    @Id
+    @Column(name = "queueId", nullable = false, updatable = false)
+    private long queueId;
+
+    @Column(name = "requestTime", nullable = true, updatable = false)
+    private Timestamp requestTime;
+
+    @Column(name = "requestType", nullable = true)
+    private String requestType;
+
+    @Column(name = "status", nullable = true)
+    private String status;
+
+    @Column(name = "startTime", nullable = true)
+    private long startTime;
+
+    @Column(name = "endTime", nullable = true)
+    private long endTime;
+
+    @Column(name = "procesingTime", nullable = true)
+    private long procesingTime;
+
+    @Column(name = "message", nullable = true)
+    private String message;
+
+    @Column(name = "documentId", nullable = true)
+    private String documentId;
+
+    @Column(name = "documentType", nullable = true)
+    private String documentType;
+
+    @Column(name = "documentPath", nullable = true)
+    private String documentPath;
+
+    @Column(name = "documentUrl", nullable = true)
+    private String documentUrl;
+
+    @Column(name = "rankView", nullable = true)
+    private int rank;
+
+    @Column(name = "view_url", nullable = true)
+    private String view_url;
+
+    @Column(name = "language", nullable = true)
+    private String language;
+
+    @Column(name = "category", nullable = true)
+    private String category;
+
+    @Column(name = "topic", nullable = true)
+    private String topic;
+
+    @Column(name = "outlinePath", nullable = true)
+    private String outlinePth;
+
+    public String getDocumentId() {
+        return documentId;
+    }
+
+    public void setDocumentId(String documentId) {
+        this.documentId = documentId;
+    }
+
+    public String getDocumentPath() {
+        return documentPath;
+    }
+
+    public void setDocumentPath(String documentPath) {
+        this.documentPath = documentPath;
+    }
+
+    public String getDocumentUrl() {
+        return documentUrl;
+    }
+
+    public void setDocumentUrl(String documentUrl) {
+        this.documentUrl = documentUrl;
+    }
+
+    public String getOutlinePth() {
+        return outlinePth;
+    }
+
+    public void setOutlinePth(String outlinePth) {
+        this.outlinePth = outlinePth;
+    }
+
+    public String getDocumentType() {
+        return documentType;
+    }
+
+    public void setDocumentType(String documentType) {
+        this.documentType = documentType;
+    }
+
+    public int getRank() {
+        return rank;
+    }
+
+    public void setRank(int rank) {
+        this.rank = rank;
+    }
+
+    public String getView_url() {
+        return view_url;
+    }
+
+    public void setView_url(String view_url) {
```

Comment:
```suggestion
    public void setViewUrl(String viewUrl) {
```
rename this to setViewUrl